common/shlibs

```diff
@@ -4062,3 +4062,5 @@ libomalloc-0.9.6.so singular-4.2.1_1
 libsingular_resources-4.2.1.so singular-4.2.1_1
 libm4ri-0.0.20200125.so m4ri-20200125_1
 libm4rie-0.0.20200125.so m4rie-20200125_1
+libkdumpfile.so.8 libkdumpfile-0.4.0_1
```

Comment:
This should be its own commit for `libkdumpfile`

Reply:
not even sure how that got in there, I have no memory of putting it there and it seem to build fine without...